system/src/Grav/Common/Assets.php

```diff
@@ -59,6 +59,8 @@ class Assets extends PropertyObject
     protected $collections;
     protected $timestamp;
 
+    /** @var array to contain next 'order' value per asset 'type', 'group' and 'position' combination */
```

Comment:
Be careful with these, the first word should be php type (as it happens to be this time).

I would add extra array: `/** @var array Array to..` just to prevent someone changing the comment and messing it up.

Reply:
I think it would also fix the comment if creating a documentation out of this:

type: array
description: to contain next...

Reply:
Fixed!